invenio_subjects_mesh/cli.py

```diff
@@ -19,13 +19,16 @@
 
 
 @click.command()
-def main():
-    """Generate new subjects_mesh.yaml file."""
+@click.argument("year")
+def main(**parameters):
+    """Generate new subjects_mesh.jsonl file."""
+    year = parameters["year"]
+
     downloads_dir = Path(__file__).parent / "downloads"
     files = MeSHDownloader(downloads_dir)
-    files.download()
+    files.download(year)
 
-    topics_reader = MeSHReader(files.topics_filepath, filter=topic_filter)
+    topics_reader = MeSHReader(files.descriptors_filepath, filter=topic_filter)
```

Comment:
I see there is a `topic_filter`, so am just checking: should `topics_reader` be renamed to `descriptors_reader`? Making sure to keep things consistent.

Reply:
The naming `topics_reader` is right here because the `MeSHReader` has been passed a `topic_filter` so that it only "reads" topics (topical MeSH terms). It doesn't read descriptors anymore, it reads this subset.